.circleci/config.yml

```diff
@@ -1375,3 +1389,50 @@ workflows:
       equal: [bump, << pipeline.parameters.action >>]
     jobs:
       - release-train
+
+  # to trigger tests manually, log into circleCI, select the project, a branch, and then click "Trigger Pipeline"
+  # and add a parameter called "run-manual-tests"
+  all-tests:
+    when:
+      and:
+        - not:
+            equal: [scheduled_pipeline, << pipeline.trigger_source >>]
+        - not: << pipeline.parameters.generate_snapshots >>
+        - not: << pipeline.parameters.generate_revenuecatui_snapshots >>
+        - or:
+            - matches: 
+                pattern: "^release/.*$"
+                value: << pipeline.git.branch >>
+            - equal:
+                - "run-manual-tests"
+                - << pipeline.parameters.action >>
+    jobs:
+      - backend-integration-tests-SK1
+      - backend-integration-tests-SK2
+      - backend-integration-tests-custom-entitlements
+      - backend-integration-tests-other
+      - build-tv-watch-and-macos
+      - build-visionos
+      - docs-build
+      - run-test-ios-14
+      - run-test-ios-15
+      - run-test-ios-16
+      - run-test-macos
+      - run-test-tvos
+      - run-test-watchos
+      - spm-custom-entitlement-computation-build
+      - spm-receipt-parser
+      - spm-release-build
+      - spm-release-build-xcode-14
+      - spm-release-build-xcode-15
+      - spm-revenuecat-ui-ios-15
+      - spm-revenuecat-ui-ios-16
+      - spm-revenuecat-ui-ios-17
+      - spm-revenuecat-ui-watchos
+      - pod-lib-lint
```

Comment:
this one is ran already in `build-test`, I see `run-test-ios-17` is not part of these jobs, so I figured you only wanted to add "the other" jobs here

Reply:
great catch